core-plugin/src/com/google/gct/idea/debugger/CloudDebugProcessState.java

```diff
@@ -227,4 +230,18 @@ public String getWaitToken() {
   public void setWaitToken(@Nullable String waitToken) {
     myWaitToken = waitToken;
   }
+
+  /**
+   * Returns whether the state is set to listen in the background
+   */
+  public boolean isListenInBackground() {
```

Comment:
I want to leave this as is. This doesn't actually enable it. It's just some state variable that tells the system that it can listen if it wants to.